Dockerfile

```diff
@@ -1,13 +1,13 @@
 # Build from source
-FROM alpine:3.12 AS Builder
+FROM bpfk/agora-builder:latest AS Builder
 ARG AGORA_VERSION="HEAD"
 WORKDIR /root/faucet/
 RUN apk add --no-cache build-base dub ldc libsodium-dev openssl-dev sqlite-dev zlib-dev
```

Comment:
Are these not in the agora-builder image already?

Reply:
Indeed